core/core.go

```diff
@@ -2,14 +2,19 @@ package core
 
 import (
 	"errors"
+	"fmt"
+	"regexp"
+	"unicode/utf8"
+
+	"github.com/gosimple/slug"
```

Comment:
If the linter will let you, move this back where it was.

Reply:
this is new though.